packages/opti/opti.1.0/opam

```diff
@@ -0,0 +1,19 @@
+opam-version: "2.0"
+name: "opti"
+version: "1.0"
+synopsis: "DSL to generate fast incremental C code from declarative specifications"
+description: "Generate C code to incrementally compute sums, products and other expressions, without recomputing them from scratch every time."
+maintainer: "Johan Magnus Jonsson <jmagnusj@gmail.com>"
+authors: ["Johan Magnus Jonsson <jmagnusj@gmail.com>"]
+license: "BSD3"
+homepage: "https://github.com/magnusjonsson/opti"
+bug-reports: "https://github.com/magnusjonsson/opti/issues"
+dev-repo: "git://github.com/magnusjonsson/opti.git"
+build: [
```

Comment:
The dependencies, including ocaml, are missing. You can look for the deps field in the other opam packages to get an idea of what it looks like